dev-db/mydumper/mydumper-0.10.7.2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_P="${PN}-$(ver_rs 3 -)"
+
+DESCRIPTION="A high-performance multi-threaded backup (and restore) toolset for MySQL"
+HOMEPAGE="https://github.com/maxbube/mydumper"
+SRC_URI="https://github.com/maxbube/${PN}/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND="app-arch/zstd
+	dev-db/mysql-connector-c:=
+	dev-libs/glib:=
+	dev-libs/libpcre:=
+	dev-libs/openssl:0=
+	sys-libs/zlib:="
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig
+	doc? ( dev-python/sphinx )"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.10.7-atomic.patch" #654314
+)
+
+S="${WORKDIR}/${MY_P}"
```

Comment:
nitpick but usually this is put right below SRC_URI without a blank line (related to it, closer to where MY_P is set, and matches skel.ebuild ordering)

Reply:
Thanks, I've missed https://github.com/gentoo/gentoo/commit/0f936ec81f1dc20fa58ba40bc17989ee44810214